brainscore/benchmarks/imagenet_c.py

```diff
@@ -0,0 +1,145 @@
+import os
+
+import numpy as np
+import pandas as pd
+import xarray as xr
+
+import brainscore
+from brainscore.benchmarks import BenchmarkBase
+from brainscore.benchmarks.trials import repeat_trials, average_trials
+from brainscore.metrics import Score
+from brainscore.metrics.accuracy import Accuracy
+from brainscore.model_interface import BrainModel
+from brainio_base.stimuli import StimulusSet
+
+BIBTEX="""@ARTICLE{Hendrycks2019-di,
```

Comment:
is there a url? the bibtex's `url` field is what we use to link to the paper from the website